### PR TITLE
Updated references to point to existing files

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

**Comment by homm** [comment]
These tutorials not related o tests. Most likely `hopper.jpg` doesn't exist in current directory as well as `lena.jpg`. I think `lena.jpg` is more recognizable name.

Also, some paths start with `Pillow/Tests/images`, other with `Tests/images`.

File: docs/handbook/tutorial.rst [hunk 1]
```diff
@@ -499,9 +499,9 @@ Reading from a tar archive
 
 ::
 
-    from PIL import TarIO
+    from PIL import Image, TarIO
 
-    fp = TarIO.TarIO("Imaging.tar", "Imaging/test/lena.ppm")
+    fp = TarIO.TarIO("Tests/images/hopper.tar", "hopper.jpg")
```
Comment (hugovk) [inline docs/handbook/tutorial.rst]: Is the second format intentionally jpg rather than ppm?

**Comment by hugovk** [comment]
These are just examples and not real code, but I approve of the change to a file which does exist in the codebase, and disassociates with sexist imagery; eg. https://www.theatlantic.com/technology/archive/2016/02/lena-image-processing-playboy/461970/.

For the paths I'm not too bothered: either leave as is; update so they'd all work from the `Pillow` directory (it's probably an unlikely use case to checkout this whole repo first); or remove paths entirely so you'd have to run it from the `images` directory, or probably more likely, move the file to your project directory.

File: docs/handbook/tutorial.rst (hunk 1 above)
Comment (wiredfool) [inline docs/handbook/tutorial.rst]: Well, it does actually work. So there's that going for it.

**Pull request merged**